integration/kubernetes/README.md

```diff
@@ -58,12 +58,14 @@ for documentation.
 Now that you have set up all that Helm needs, you can use the helper script `helm-generate.sh` to help you quickly generate the templates(overwriting the existing ones).
 The script takes only one argument, the target setup name.
 
+> Note: `helm-generate.sh` uses `helm template` command under the hood.
+Templates ignored with `.helmignore` file will not be generated.  
+
```

Comment:
Added ref where contents added to `.helmignore` in Helm deployment will affect kubectl deployment